contrib/internal/fasthttptrace/fasthttpheaderscarrier.go

```diff
@@ -11,18 +11,18 @@ import (
 	"github.com/valyala/fasthttp"
 )
 
-// FastHTTPHeadersCarrier implements tracer.TextMapWriter and tracer.TextMapReader on top
+// HTTPHeadersCarrier implements tracer.TextMapWriter and tracer.TextMapReader on top
```

Comment:
@mtoffl01 Why is this kept in a separate package?

Reply:
I figured it'd make sense to "section" it off into its own "carrier" package rather than keep it within the fashttp contrib pkg in case we add support for other libraries built on fasthttp.

Reply:
Here is the PR if that provides any context: https://github.com/DataDog/dd-trace-go/pull/2218

Reply:
@mtoffl01 Ok, that makes sense. Thanks!